docs/dev/table/connect.md

```diff
@@ -276,6 +276,40 @@ tables:
         type: VARCHAR
 {% endhighlight %}
 </div>
+
+<div data-lang="DDL" markdown="1">
+{% highlight sql %}
+-- CREATE a 010 version Kafka table start from the earliest offset(as table source) and append mode(as table sink).
+CREATE TABLE MyUserTable (
+  `user` bigint,
```

Comment:
Please use capital letters for all SQL keyword including data types. It makes it easier for readers to distinguish between fields/tables and important concepts.

Reply:
Already use capital letters ?

Reply:
All the keywords should use capital letters including types.